includes/Story_Post_Type.php

```diff
@@ -329,6 +330,7 @@ public static function admin_enqueue_scripts( $hook ) {
 					'capabilities'     => [
 						'hasPublishAction'      => $has_publish_action,
```

Comment:
I wonder if it might make sense to rename these actions to something more clear now when we're not using what comes from WP?
E.g. `canPublish` instead of `hasPublishAction` and `canUploadMedia` instead of `hasUploadMediaAction`, etc.

Thoughts?

Reply:
Not in scope of this PR. I didn't change the name of these variables as they are used as well.